include/deal.II/numerics/vector_tools.templates.h

```diff
@@ -234,74 +236,124 @@ namespace VectorTools
         function.vector_value_list(generalized_support_points,
                                    function_values);
 
-        // FIXME: In case of an FESystem we have to apply this
-        // transformation according to the conformity of each base element.
+        // A small helper function to transform a component range starting
+        // at offset from the real to the unit cell according to the
+        // supplied conformity.
+        //
+        // FIXME: This should be refactored into the mapping (i.e.
+        // implement the inverse function of Mapping<dim,
+        // spacedim>::transform). Further, the finite element should make
+        // the information about the correct mapping directly accessible -
+        // fe.conforming_space is not the right call (thing about BDM).
```

Comment:
thing -> think

I don't understand the comment about the BDM element (or the comment in general). Is your complaint that we can't look at this at the element level, but need to consider it per-shape function?

Reply:
This was just meant as an example for which we implement a special transformation (whether needed or not is another question). I have reworded the comment to:
```
         // FIXME: This should be refactored into the mapping (i.e.               
         // implement the inverse function of Mapping<dim, spacedim>::transform). 
         // Further, the finite element should make the information about         
         // the correct mapping directly accessible (i.e. which MappingType       
         // should be used). Using fe.conforming_space might be a bit of a        
         // problem because we only support Lagrange elements as well as,
         // Hcurl, and Hdiv conforming mappings. 
```